Library/Formula/fpp.rb

```diff
@@ -0,0 +1,20 @@
+class Fpp < Formula
+  homepage "https://facebook.github.io/PathPicker/"
+  url "https://facebook.github.io/PathPicker/dist/fpp.0.5.3.tar.gz"
+  sha256 "94b77ef10a128a694f6302ce578a2d07a3fd2892299d341b22be9496abd7277d"
+  head "https://github.com/facebook/pathpicker.git"
```

Comment:
I presume this repo is being kept private until release? Just checking the link isn't dead/mistyped/etc.